amrit-docker-setup/docker-compose.yml

```diff
@@ -0,0 +1,514 @@
+    networks:
+      - amrit-network
+    volumes:
+      - ./logs/fhir-api:/usr/local/tomcat/logs
+    depends_on:
+      - mysql
+    env_file:
+      - ./.env
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--no-verbose",
+          "--tries=1",
+          "--spider",
+          "http://localhost:8080/v3/api-docs",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  flw-api:
+    build:
+      context: ./API/FLW-API
+      dockerfile: Dockerfile
+    container_name: flw-api-container
+    ports:
+      - "8081:8080"
+    networks:
+      - amrit-network
+    volumes:
+      - ./logs/flw-api:/usr/local/tomcat/logs
+    depends_on:
+      - mysql
+    env_file:
+      - ./.env
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--no-verbose",
+          "--tries=1",
+          "--spider",
+          "http://localhost:8080/v3/api-docs",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  helpline104-api:
+    build:
+      context: ./API/Helpline104-API
+      dockerfile: Dockerfile
+    container_name: helpline104-api-container
+    ports:
+      - "8091:8080"
+    networks:
+      - amrit-network
+    volumes:
+      - ./logs/helpline104-api:/usr/local/tomcat/logs
+    depends_on:
+      - mysql
+    env_file:
+      - ./.env
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--no-verbose",
+          "--tries=1",
+          "--spider",
+          "http://localhost:8080/v3/api-docs",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  helpline1097-api:
+    build:
+      context: ./API/Helpline1097-API
+      dockerfile: Dockerfile
+    container_name: helpline1097-api-container
+    ports:
+      - "8090:8080"
+    networks:
+      - amrit-network
+    volumes:
+      - ./logs/helpline1097-api:/usr/local/tomcat/logs
+    depends_on:
+      - mysql
+    env_file:
+      - ./.env
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--no-verbose",
+          "--tries=1",
+          "--spider",
+          "http://localhost:8080/v3/api-docs",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  beneficiaryid-generation-api:
+    build:
+      context: ./API/BeneficiaryID-Generation-API
+      dockerfile: Dockerfile
+    container_name: beneficiaryid-generation-api-container
+    ports:
+      - "8092:8080"
+    networks:
+      - amrit-network
+    volumes:
+      - ./logs/beneficiaryid-generation-api:/usr/local/tomcat/logs
+    depends_on:
+      - mysql
+    env_file:
+      - ./.env
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "wget",
+          "--no-verbose",
+          "--tries=1",
+          "--spider",
+          "http://localhost:8080/v3/api-docs",
+        ]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+  # Database service
+  mysql:

```

Comment:
Infra components are already handled in amrit local setup.
Modifying that will suffice. 
Let's keep infra docker compose and app docker compose separate.